### DISCUSSION TITLE
decoding

### DISCUSSION THREAD

**Discussion by Swift-Sand** [wierkstudio/ciphereditor#35] [Ideas]
The thing that makes this project not my go-to for cryptology is the fact that you can't decrypt messages, I think making this your one-stop-shop would make this software the top thing, that's my only complaint. thank you devs, this project is amazing. :-)

**Comment by ffraenz** [comment]
Most operations on ciphereditor come with support for decoding/undoing a translation (the hash function is one exception). The data flow decides on wether a value gets encoded or decoded. Feed a value into the "Encoded data" control and it gets decoded to the opposite control. A general "decrypt this piece of data" feature is not implemented, right now.

**Reply by Swift-Sand** [reply]
Thank you man. guess i wasent doing it right.
